recipes/flint/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

`conan_basic_setup` without `TARGETS` will call `include_directories`
```suggestion
conan_basic_setup(TARGETS)
```

---

nice catch, thanks!